### DISCUSSION TITLE
blog/2024/11/01/the%20few%20opportunities%20for%20slacking%20off

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#28] [General]
# blog/2024/11/01/the%20few%20opportunities%20for%20slacking%20off

只剩下最后2个月的摸鱼时间了。

https://996.ninja/blog/2024/11/01/the%20few%20opportunities%20for%20slacking%20off

**Comment by Sorrylooo** [comment]
全英就是好，反正我们老板看不懂

**Reply by Sorrylooo** [reply]
Play scary maze game这个好玩

**Reply by MingGH** [reply]
是的，朋友玩过感动的都哭了
